modules/virt-live-migration-metrics.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-monitor-vmi-migration.adoc
+// * virt/logging_events_monitoring/virt-prometheus-queries.adoc
+
+:_content-type: REFERENCE
+[id="virt-live-migration-metrics_{context}"]
+= Live migration metrics
+
+The following metrics can be queried to show live migration status:
+
+`kubevirt_migrate_vmi_data_processed_bytes`:: The amount of guest operating system (OS) data that has migrated to the new virtual machine (VM). Type: Gauge.
+
+`kubevirt_migrate_vmi_data_remaining_bytes`:: The amount of guest OS data that remains to be migrated. Type: Gauge.
+
+`kubevirt_migrate_vmi_dirty_memory_rate_bytes`:: The rate at which memory is becoming dirty in the guest OS. Dirty memory is data that has been changed but not yet written to disk. Type: Gauge.
+
+`kubevirt_migrate_vmi_pending_count`:: The number of pending migrations. Type: Gauge.
+
+`kubevirt_migrate_vmi_scheduling_count`:: The number of scheduling migrations. Type: Gauge.
+
+`kubevirt_migrate_vmi_running_count`:: The number of running migrations. Type: Gauge.
+
+`kubevirt_migrate_vmi_succeeded`:: The number of successfully completed migrations. Type: Gauge.
```

Comment:
if I remember correctly - it was renamed to `kubevirt_migrate_vmi_succeeded_total`

Reply:
Resolved; I will open a follow-up PR to rename these again for 4.12.1.